masschallenge/django-accelerator
# [AC-9014]: Add organisation note model

[AC-9014](https://masschallenge.atlassian.net/browse/AC-9014)

How to test:
- Ensure all the requirements in [AC-9014](https://masschallenge.atlassian.net/browse/AC-9014) are satisfied
